python/apps/taiga/src/taiga/workflows/api/validators.py

```diff
@@ -23,3 +23,7 @@ class WorkflowStatusValidator(BaseModel):
 
 class UpdateWorkflowStatusValidator(BaseModel):
     name: Name | None
+
+
+class DeleteWorkflowStatusQuery(BaseModel):
```

Comment:
why Query instead of Validator?

Reply:
Because OpenAPI is removing support and doesn't encourage to send parameters in the Body of a Delete request. It should be sent in the url as a query param. I can look for the link I reviewed if you're interested.

Reply:
I meant the name, why everything is a "XXXValidator" and this is a "XXXXQuery"?

Reply:
Because validator's params normally come from a body request, and the traditional query param are documented in the same api file.

There exist other cases of XXXXQuery (pagination, ordering) representing query params, and they are separated from the validators.

But you're right they are also validators, that's why I find it better not to move it to another file. The ideal would be perhaps to combine the two approaches (xxxxxQueryValidator), but this would become too large.